pkgs/development/nim-packages/build-nim-package/default.nix

```diff
@@ -0,0 +1,56 @@
+{ lib, stdenv, nim }:
+
+let
+  nim_builder = stdenv.mkDerivation {
```

Comment:
do you think it is worth exposing it in the nimPackages scope to make it overrideable?

Reply:
Yes, it will probably need some adjustsments in the future.

Reply:
fair enough